CHANGELOG.md

```diff
@@ -2,14 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## TS CLI&SDK [2.1.0](https://github.com/marinade-finance/validator-bonds/compare/v2.0.6...v2.1.0) (2025-01-31)
+## TS CLI&SDK [2.1.2](https://github.com/marinade-finance/validator-bonds/compare/v2.1.1...v2.1.2) (2025-02-04)
```

Comment:
(non-blocking)
There is now no description of this new version (it previously said "Fix of wrong npm registry deployment" in this PR), not sure if intended.